artcommon/artcommonlib/util.py

```diff
@@ -430,3 +430,17 @@ def detect_package_managers(metadata, dest_dir: Path):
         if any(dest_dir.joinpath(file).is_file() for file in files):
             pkg_managers.append(pkg_manager)
     return pkg_managers
+
+
+def get_konflux_network_mode(metadata):
```

Comment:
Thinking this method should be put in Metadata class.

Reply:
Thanks - updated